runtime/tr.source/trj9/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -18739,12 +18742,22 @@ J9::Z::TreeEvaluator::pd2lVariableEvaluator(TR::Node* node, TR::CodeGenerator* c
    generateRIInstruction(cg, TR::InstOpCode::AGHI, node, lengthReg, 0x1);
```

Comment:
The commit says this has been fixed, but this line is still present. I think you may have forgot to remove it.

Reply:
Fixed.